src/components/ShortcutLink.jsx

```diff
@@ -52,16 +44,16 @@ export const ShortcutLink = ({
           display={display}
           description={description}
           IconContent={
-            <img
-              src={
-                iconMimeType
-                  ? `data:${iconMimeType};base64,${icon}`
-                  : `data:image/svg+xml;base64,${window.btoa(icon)}`
-              }
-              width={computedSize}
-              height={computedSize}
-              alt={filename}
-            />
+            <div className="u-w-2 u-h-2">
```

Comment:
nit: is this div mandatory? Can we put the className on `img`?

Reply:
It was to do exactly like on SquareAppIcon to avoid issues caused by differences.